### PR TITLE
Update `AboutUs.md` and add PPP for marcuspang

### PR DESCRIPTION
**Pull request by marcuspang**
# Changes

- Add portfolio page markdown page 
- Add profile photo
- Fix some styling issues for the template portfolio page

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
-layout: page
-title: About Us
+layout: page title: About Us
```
Comment (marcuspang) [inline docs/AboutUs.md:2]:
```suggestion
layout: page 
title: About Us
```

Not sure why my IDE auto-formatted this section

File: docs/team/devanshshah1309.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
 ---
-layout: page
-title: Devansh Shah's Project Portfolio Page
+layout: page title: Devansh Shah's Project Portfolio Page
```
[the remark this answers is not in the document]
Comment (marcuspang) [reply docs/team/devanshshah1309.md:2]: Yep

Comment (marcuspang) [inline docs/team/devanshshah1309.md:2]:
```suggestion
layout: page
title: Devansh Shah's Project Portfolio Page
```

File: docs/team/marcuspang.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: page title: Marcus Pang's Project Portfolio Page
```
Comment (Devanshshah1309) [inline docs/team/marcuspang.md:2]:
```suggestion
layout: page
title: Marcus Pang's Project Portfolio Page
```

**Review by Devanshshah1309 — APPROVED** [review]

**Pull request merged**